assets/multiqc_config.yaml

```diff
@@ -1,12 +1,12 @@
 report_comment: >
-    This report has been generated by the <a href="https://github.com/nf-core/cutandrun" target="_blank">nf-core/cutandrun</a>
-    analysis pipeline. For information about how to interpret these results, please see the
-    <a href="https://nf-co.re/cutandrun" target="_blank">documentation</a>.
+  This report has been generated by the <a href="https://github.com/nf-core/cutandrun" target="_blank">nf-core/cutandrun</a>
```

Comment:
file should be `assets/multiqc_config.yml`

Reply:
Done